src/macaron/database/table_definitions.py

```diff
@@ -287,7 +287,7 @@ class Repository(ORMBase):
     #: The path to the repo on the file system.
     fs_path: Mapped[str] = mapped_column(String, nullable=False)
 
-    def __init__(self, *args, files: list[str] | None = None, **kwargs):  # type: ignore[no-untyped-def]
+    def __init__(self, *args: Any, files: list[str] | None = None, **kwargs: Any):
```

Comment:
I don't think `Any` is better than `# type: ignore` here. What I didn't know at the time I suppressed the type error here was that it's recommended to handle `args` and `kwargs` using `ParamSpec`: https://docs.python.org/3/library/typing.html#typing.ParamSpec

Reply:
It's slightly better in that at least it is requiring the `files` argument to be typed (which it already was).
I hadn't seen `ParamSpec`, but I think it is mainly relevant in expressing forwarding args/kwargs to a passed callable where the function is generic on the callable's signature, which is not the case here, where they are being passed to the superclass __init__ whose types are fixed, so args/kwargs should simply be typed with the correct element type, which in this case cannot be more precise than Any because the superclass signature is `def __init__(self, **kw: Any)`.

Reply:
> which in this case cannot be more precise than Any because the superclass signature is `def __init__(self, **kw: Any)`.

Huh! I had assumed that the `__init__` function in the `DeclarativeBase` class would use type parameters, but maybe that's tricky because `ParamSpec` was introduces in Python 3.10 and SQLAlchemy needs to be compatible with older Python versions, hence using `Any`.